examples/access-controllable-examples/README.md

```diff
@@ -0,0 +1,450 @@
+   to the `<ALICE_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet` as `<ALICE_ACCOUNT>`, and to the `<BOB_ACCOUNT_NAME>.<MASTER_ACCOUNT_NAME>.testnet` as `<BOB_ACCOUNT>` for simplicity.
+
+2. **Compile Contract to wasm file**
+   For compiling the contract, just run the `access_controllable_base/build.sh` script. The target file with compiled contract will be `../target/wasm32-unknown-unknown/release/access_controllable_base.wasm`
+
+   ```shell
+   $ cd access_controllable_base
+   $ ./build.sh
+   $ cd ..
+   ```
+
+3. **Deploy and init a contract**
+   ```shell
+   $ near deploy --accountId <CONTRACT_ACCOUNT> --wasmFile ../target/wasm32-unknown-unknown/release/access_controllable_base.wasm --initFunction new --initArgs '{}'
+   ```
+
+## Example of using the contract with access control plugin
+### Calling the methods with access control
+For using the method, `level_a_incr` you should be a member of GroupA.  Alice is not a member of any group, so she can't use this method.
+
+```shell
+$ near call <CONTRACT_ACCOUNT> level_a_incr --accountId <ALICE_ACCOUNT>
+ERROR
+$ near view get_counter
+0
+```
+
+Let's make Alice a member of GroupA.
+```shell
+$ near call <CONTRACT_ACCOUNT> acl_grant_role '{"role": "GroupA", "account_id": "<ALICE_ACCOUNT>"}' --accountId <CONTRACT_ACCOUNT>
+```
+
+Now Alice is a member of GroupA and can call the `level_a_incr` method
+```shell
+$ near call <CONTRACT_ACCOUNT> level_a_incr --accountId <ALICE_ACCOUNT>
+$ near view get_counter
+1
+```
+
+As well as calls the `level_ab_incr` method, which allowed for both GroupA and GroupB members.
+```shell
+$ near call <CONTRACT_ACCOUNT> level_ab_incr --accountId <ALICE_ACCOUNT>
+$ near view get_counter
+2
+```
+
+### Admin of the group not a member of the group
+Note, the admin of the group may not be a member of this group. For example, the `<CONTRACT_ACCOUNT>` is a super admin, but he
+can't execute the `level_a_incr` method. 
+
+```shell
+$ near view get_counter
+2
+$ near call <CONTRACT_ACCOUNT> level_a_incr --accountId <CONTRACT_ACCOUNT>
+ERROR
+$ near view get_counter
+2
+```
+
+### A few super admins
+There may be two or more super admins. 
+By default, there are no interfaces that allow you to add or remove super admins. 
+You can add the corresponding functions to the contract as follows:
+
+```rust
+/// method for adding new super admin
+pub fn add_super_admin(&mut self, new_super_admin_account_id: &AccountId) {
+    ::near_sdk::require!(self.__acl.is_super_admin(&near_sdk::env::predecessor_account_id()),
+                         "Method can be run only by super admin");
+    self.__acl.add_super_admin_unchecked(new_super_admin_account_id); 
+}
+
+/// method for removing super admin
+pub fn remove_super_admin(&mut self, super_admin_account_id: &AccountId) {
+   ::near_sdk::require!(self.__acl.is_super_admin(&near_sdk::env::predecessor_account_id()),
```

Comment:
```suggestion
   ::near_sdk::require!(self.acl_is_super_admin(near_sdk::env::predecessor_account_id()),
```

Reply:
fix https://github.com/aurora-is-near/near-plugins/pull/34/commits/1a7a34b88d2b4513d1de096b408ef5fc17e46209